lib/containers/container_images.pm

```diff
@@ -32,6 +33,7 @@ sub build_container_image {
 
     die 'Argument $image not provided!'   unless $image;
     die 'Argument $runtime not provided!' unless $runtime;
+    return 0 if is_sle_container_supported;
```

Comment:
The image can be non-sle. This line should not be here but on `tests/containers/docker_image.pm` and on `tests/containers/podman_image.pm` and there should be also `record_info` so the test reviewer know what happened.

Reply:
it was addressed as well.